bigchaindb/core.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, host=None, port=None, dbname=None,
         self.host = host or bigchaindb.config['database']['host']
         self.port = port or bigchaindb.config['database']['port']
         self.dbname = dbname or bigchaindb.config['database']['name']
+        self.backend = backend or get_backend()
```

Comment:
Ok, also note that if you need to access the `connection` from a `bigchain` instance you can always do `self.backend.connection`